accelerator/cli_info.h

```diff
@@ -83,14 +84,15 @@ static struct ta_cli_argument_s {
     {"milestone_depth", optional_argument, NULL, MILESTONE_DEPTH_CLI, "IRI milestone depth"},
     {"mwm", optional_argument, NULL, MWM_CLI, "minimum weight magnitude"},
     {"seed", optional_argument, NULL, SEED_CLI, "IOTA seed"},
-    {"cache", no_argument, NULL, CACHE, "Enable cache server"},
+    {"cache", no_argument, NULL, CACHE_ACQUIRE, "Enable cache server"},
```

Comment:
Using plain `CACHE` would be straightforward.

Reply:
Will cache server be enabled by default?

Reply:
Now it is disabled by default

Reply:
Option `cache` might be misleading for users who expect to switch between `--cache on` and `--cache off`.

Reply:
Should I name it as `cache_on`?

Reply:
Alternatively, `--cache` can accept necessary argument, which indicates cache on/off.